### PR TITLE
Updated Makefile and build.sbt

### PR DESCRIPTION
**Pull request by jkbradley**
to make packaging for spark packages easier

### CODE REVIEW

**Comment by jkbradley** [comment]
No hurry to merge this

**Comment by mengxr** [comment]
Btw, the build doesn't work with Spark master, which uses Scala 2.11 by default. We might need to cross build this with sbt.

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ sparkVersion := sys.props.getOrElse("spark.version", "1.4.1")
 spName := "graphframes/graphframes"
 
 // Don't forget to set the version
-version := "0.1.0-SNAPSHOT"
+version := "0.1.0-spark" + sparkVersion.value.substring(0, 3)
```
Comment (mengxr) [inline build.sbt]: If the commit is not for the release, we should change the version back to `-SNAPSHOT`.

Comment (jkbradley) [inline build.sbt]: How would you recommend we specify that?

**Comment by mengxr** [comment]
LGTM. I don't have good recommendations, but I can try some ideas after merge:)

**Pull request merged**